.github/workflows/reusable-build-test-release.yml

```diff
@@ -369,13 +369,23 @@ jobs:
       requirement_test: ${{ steps.testset.outputs.requirement_test }}
       scripted_inputs: ${{ steps.testset.outputs.scripted_inputs }}
       escu: ${{ steps.testset.outputs.escu }}
+      old_modinput_functional: ${{ steps.modinput-version.outputs.old_modinput_tests }}
```

Comment:
Avoid names like 'old' or 'new'. When newer version is needed it ends with 'older', 'newer', 'older2' etc. names

Reply:
Agree - resolved